tee-worker/enclave-runtime/src/rpc/worker_api_direct.rs

```diff
@@ -218,9 +218,9 @@ where
 		Ok(json!(json_value))
 	});
 
-	// state_getEnclave
-	let state_get_enclave_name: &str = "state_getEnclave";
-	io.add_sync_method(state_get_enclave_name, |_: Params| {
+	// state_getMrenclave
+	let state_get_mrenclave_name: &str = "state_getMrenclave";
+	io.add_sync_method(state_get_mrenclave_name, |_: Params| {
```

Comment:
I think we should rename `state_updateScheduledEnclave` -> `state_updateScheduledMrenclave` also.

Reply:
Updating "enclave" makes sense too as it basically says: I'd like to update the enclave schedule, only enclave with _this_ mrenclave can produce blocks start from _this_ block number, so we schedule the enclave behaviour as a whole, which includes both mrenclave and sidechain_block_number as criteria.

I'm fine with either way though if you insist on renaming

Reply:
Ok, i see the difference 🙃